guides/introduction/overview.md

```diff
@@ -6,11 +6,37 @@ Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin nibh augue, susci
 
 ## Tools
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin nibh augue, suscipit a, scelerisque sed, lacinia in, mi. Cras vel lorem. Etiam pellentesque aliquet tellus. Phasellus pharetra nulla ac diam. Quisque semper justo at risus. Donec venenatis, turpis vel hendrerit interdum, dui ligula ultricies purus, sed posuere libero dui id orci. Nam congue, pede vitae dapibus aliquet, elit magna vulputate arcu, vel tempus metus leo non est. Etiam sit amet lectus quis est congue mollis. Phasellus congue lacus eget neque. Phasellus ornare, ante vitae consectetuer consequat, purus sapien ultricies dolor, et mollis pede metus eget nisi. Praesent sodales velit quis augue. Cras suscipit, urna at aliquam rhoncus, urna quam viverra nisi, in interdum massa nibh nec erat.
+Depending on where you are at in your programming journey here are some additional tools you may want to consider installing.
+
+### Text Editors
+
+A text editor is at the core of a developer's toolbox, so we shoud choose it carefully.
```

Comment:
A text editor is the core of every developer's toolbox, that's why it's important to choose it wisely.